docs/howtos/Release_Process_with_PR_Rebase.md

```diff
@@ -0,0 +1,35 @@
+# Release Process with PR Rebase
+
+Rebasing your pull requests keeps your commit history clean and integrates changes from the main branch, ensuring a smoother merge.
+
+**Note:** the current release is tagged as `release-v*`. Replace `release-v*` with the actual version.
+
+#### Before creating a new local working branch:
+   - Make sure you have local versions of the Learning Equality `develop` branch and the Learning Equality `release-v*` branch.
```

Comment:
> the Learning Equality `develop` branch and the Learning Equality `release-v*` branch

To keep the instructions succinct and to the point, there's no need to reference Learning Equality here.